spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/zipkin/ZipkinAutoConfiguration.java

```diff
@@ -56,10 +51,4 @@ PropertiesZipkinConnectionDetails zipkinConnectionDetails(ZipkinProperties prope
 		return new PropertiesZipkinConnectionDetails(properties);
 	}
 
-	@Bean
-	@ConditionalOnMissingBean
-	public BytesEncoder<Span> spanBytesEncoder() {
```

Comment:
removing this from top-level because brave uses `BytesEncoder<MutableSpan>`, not `BytesEncoder<Span>` which pulls in zipkin core types. We can push this into the otel config if desired as it still requires the core zipkin jar, but let's be sure to not accidentally add that dep for normal brave.